Deepak_Yadav/p_node/schemas/employee.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+	Name:{
```

Comment:
Why do you have variable names starting with uppercase???

Reply:
Code updated...